h5py/_hl/group.py

```diff
@@ -89,7 +89,10 @@ def create_dataset(self, name, shape=None, dtype=None, data=None, **kwds):
 
         maxshape
             (Tuple or int) Make the dataset resizable up to this shape. Use None for
-            axes you want to be unlimited. Integers can be used for 1D shape.
+            axes within the tuple you want to be unlimited. Integers can be used for 1D shape.
+            For 1D datsets with quasi-unlimited maxshape, a shape tuple of length 1 must be
```

Comment:
```suggestion
            For 1D datsets with unlimited maxshape, a shape tuple of length 1 must be
```

I know 'quasi-unlimited' is technically more accurate, but I think it's a distraction from the point we're trying to make. No-one is assuming that passing `(None,)` somehow gives you infinite storage. :wink: